packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -1,12 +1,350 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
@chrismaree Best to think of this as a child contract. It will be deployed by the `BridgePoolFactory` and only stores a reference to the parent. It reads global state variables from the parent via external getter methods.

Reply:
sounds good.